README.md

```diff
@@ -231,24 +232,27 @@ These are not blockers for any of the core sequence and so they are worked into
 1. [Diversity, Inclusion & Implicit Bias](/diversity-inclusion-bias/inclusion.md)
 1. [Tech Tips and Rules](https://docs.google.com/document/d/1SWiUl0lYQ0sYDM8vQnVhJVOvT4MRA-9pDglKOgqTWak/edit)
 1. [Giving Presentations](giving-presentations/giving-presentations.md)
-1. [Whiteboarding](/career/interviewing/whiteboarding.md)
-1. [Whiteboarding Interview Challenges](/career/interviewing/week-3-whiteboard-challenges.md)
+1. [Whiteboarding](/interviewing/whiteboarding.md)
```

Comment:
why did you change these?  The files are definitely still in the career/ folder.